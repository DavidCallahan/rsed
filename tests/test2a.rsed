foreach all
   if match "more" then
       stop
   end
   print
end
print "not reached"
