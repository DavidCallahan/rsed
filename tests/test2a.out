1
2
3 more
